src/richie/apps/search/viewsets/categories.py

```diff
@@ -60,7 +59,7 @@ def list(self, request, version, kind):
             "meta": {
                 "count": len(query_response["hits"]["hits"]),
                 "offset": offset,
-                "total_count": query_response["hits"]["total"],
+                "total_count": query_response["hits"]["total"]["value"],
```

Comment:
this does not seem related to the doc type commit?

Reply:
I put this in a separate commit

Reply:
:+1: